diff -up bitcoin-0.14.1/qa/rpc-tests/test_framework/util.py.orig bitcoin-0.14.1/qa/rpc-tests/test_framework/util.py
--- bitcoin-0.14.1/qa/rpc-tests/test_framework/util.py.orig	2017-04-23 03:33:36.762009639 +0000
+++ bitcoin-0.14.1/qa/rpc-tests/test_framework/util.py	2017-04-23 18:59:36.276204684 +0000
@@ -35,7 +35,7 @@ PORT_MIN = 11000
 # The number of ports to "reserve" for p2p and rpc, each
 PORT_RANGE = 5000
 
-BITCOIND_PROC_WAIT_TIMEOUT = 60
+BITCOIND_PROC_WAIT_TIMEOUT = 3600
 
 
 class PortSeed:
