diff -up bitcoin-0.14.0.orig/src/wallet/rpcwallet.cpp bitcoin-0.14.0/src/wallet/rpcwallet.cpp
--- bitcoin-0.14.0.orig/src/wallet/rpcwallet.cpp	2017-03-05 18:09:49.000000000 +0000
+++ bitcoin-0.14.0/src/wallet/rpcwallet.cpp	2017-03-08 21:05:03.964571932 +0000
@@ -6,6 +6,7 @@
 #include "amount.h"
 #include "base58.h"
 #include "chain.h"
+#include "coincontrol.h"
 #include "consensus/validation.h"
 #include "core_io.h"
 #include "init.h"
@@ -361,9 +362,14 @@
     std::string strError;
     vector<CRecipient> vecSend;
     int nChangePosRet = -1;
+    CCoinControl coinControl;
+    if (GetArg("-destchange", "") != "") {
+        CBitcoinAddress addr = GetArg("-destchange", "");
+        coinControl.destChange = addr.Get();
+    }
     CRecipient recipient = {scriptPubKey, nValue, fSubtractFeeFromAmount};
     vecSend.push_back(recipient);
-    if (!pwalletMain->CreateTransaction(vecSend, wtxNew, reservekey, nFeeRequired, nChangePosRet, strError)) {
+    if (!pwalletMain->CreateTransaction(vecSend, wtxNew, reservekey, nFeeRequired, nChangePosRet, strError, &coinControl)) {
         if (!fSubtractFeeFromAmount && nValue + nFeeRequired > curBalance)
             strError = strprintf("Error: This transaction requires a transaction fee of at least %s", FormatMoney(nFeeRequired));
         throw JSONRPCError(RPC_WALLET_ERROR, strError);
@@ -979,7 +985,12 @@
     CAmount nFeeRequired = 0;
     int nChangePosRet = -1;
     string strFailReason;
-    bool fCreated = pwalletMain->CreateTransaction(vecSend, wtx, keyChange, nFeeRequired, nChangePosRet, strFailReason);
+    CCoinControl coinControl;
+    if (GetArg("-destchange", "") != "") {
+        CBitcoinAddress addr = GetArg("-destchange", "");
+        coinControl.destChange = addr.Get();
+    }
+    bool fCreated = pwalletMain->CreateTransaction(vecSend, wtx, keyChange, nFeeRequired, nChangePosRet, strFailReason, &coinControl);
     if (!fCreated)
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, strFailReason);
     CValidationState state;
