diff -up bitcoin-0.17.0/test/functional/test_runner.py.orig bitcoin-0.17.0/test/functional/test_runner.py
--- bitcoin-0.17.0/test/functional/test_runner.py.orig	2018-10-01 10:04:33.000000000 +0000
+++ bitcoin-0.17.0/test/functional/test_runner.py	2018-10-03 16:28:55.955394038 +0000
@@ -230,7 +230,7 @@
     logging.basicConfig(format='%(message)s', level=logging_level)
 
     # Create base test directory
-    tmpdir = "%s/test_runner_₿_🏃_%s" % (args.tmpdirprefix, datetime.datetime.now().strftime("%Y%m%d_%H%M%S"))
+    tmpdir = "%s/test_runner_%s" % (args.tmpdirprefix, datetime.datetime.now().strftime("%Y%m%d_%H%M%S"))
     os.makedirs(tmpdir)
 
     logging.debug("Temporary test directory at %s" % tmpdir)
